tests/resources/algorithms/logistic_regression.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2022 Ocean Protocol Foundation
+# SPDX-License-Identifier: Apache-2.0
+#
+"""
+=========================================================
+Logistic Regression 3-class Classifier
+=========================================================
+
+Show below is a logistic-regression classifiers decision boundaries on the
+first two dimensions (sepal length and width) of the `iris
+<https://en.wikipedia.org/wiki/Iris_flower_data_set>`_ dataset. The datapoints
+are colored according to their labels.
+
+"""
+
+# Code source: Gaël Varoquaux
+# Modified for documentation by Jaques Grobler
+# Modified by Ocean Protocol Foundation
+# License: BSD 3 clause
+
+import json
+import os
+import pickle
+import sys
+
+# Uncomment if running locally. g not included in oceanprotocol/algo_dockers:python-panda image
+# import matplotlib.pyplot as plt
```

Comment:
I chose not to update the [python-panda docker image requirements.txt] (https://github.com/oceanprotocol/algo_dockers/blob/main/python-panda/requirements.txt) to include `matplotlib` because I don't know if I have the authority to push a new image to docker hub.

Reply:
PR for adding `matplotlib` to `python-panda` docker image was created.  Once merged, these lines can be uncommented. https://github.com/oceanprotocol/algo_dockers/pull/4